src/components/UserHeader.js

```diff
@@ -103,6 +107,8 @@ UserHeader.propTypes = {
   isSameUser: PropTypes.bool,
   hasCover: PropTypes.bool,
   onSelect: PropTypes.func,
+  isPopoverVisible: PropTypes.bool,
```

Comment:
Can we move this prop above functions? Same for defaultProps.

Reply:
Copy. will move.